src/features/user/ManageProjects/components/DetailedAnalysis.tsx

```diff
@@ -216,10 +213,30 @@ export default function DetailedAnalysis({ handleBack, handleNext }: Props): Rea
                     </div>
                 </div>
 
-                <div className={styles.formField}>
+                <div className={styles.formFieldLarge}>
                     <p className={styles.inlineLinkButton}>Add another cerification</p>
                 </div>
 
+                <div className={styles.formField}>
+                    <div className={`${styles.formFieldHalf}`}>
+                        <AnimatedButton
+                            onClick={handleBack}
+                            className={styles.secondaryButton}
+                        >
+                          <BackArrow/>
+                            <p>Back to project media</p>
```

Comment:
Please do not forgot to add translation resources here and everywhere else for this patch.